backends/models.py

```diff
@@ -0,0 +1,35 @@
+"""
+OAuth token model
+"""
+import datetime
+
+import pytz
+from django.db import models
+from django.db.models import TextField, CharField, DateTimeField
+
+
+class OAuthToken(models.Model):
+    """
+    Store OAuth access and refresh tokens for an external API, such as FluidReview,
+    that has short-lived refresh tokens (invalidated whenever used).
+    """
+    api = CharField(max_length=30, unique=True, primary_key=True, null=False)
```

Comment:
I'm not sure it makes sense to have this model support multiple APIs. The only credentials we will support for the forseeable future is FluidReview credentials. Also, the social auth oauth credentials are stored elsewhere already.

Reply:
Will move everything to a `fluidreview` app